web/themes/joinup/templates/block/block--toc-outline.twig

```diff
@@ -0,0 +1,31 @@
+{#
+/**
+ * @file
+ * Theme override to display a block.
```

Comment:
Let's update this to explain why we override the template, like we did for the other template.